tests/unit/test_main.py

```diff
@@ -101,15 +101,86 @@ def test_verify_api_urls(self):
         urls = get_napp_urls(self.napp)
         self.assertEqual(expected_urls, urls)
 
+    @patch('napps.kytos.topology.main.StoreHouse.get_data')
+    def test_restore_network_status(self, mock_storehouse):
+        """Test restore_network_status."""
+        dpid = '00:00:00:00:00:00:00:01'
+        mock_switch = get_switch_mock(dpid)
+        mock_interface = create_autospec(Interface)
+        mock_switch.interfaces = {1: mock_interface}
+        self.napp.controller.switches = {dpid: mock_switch}
+        msg_expected = 'There is no status saved to restore.'
+        msg_success = 'Administrative status restored.'
+        status = {
+            'network_status': {
+                'id': 'network_status',
+                'switches': {
+                    '00:00:00:00:00:00:00:01': {
+                        'dpid': '00:00:00:00:00:00:00:01',
+                        'enabled': True,
+                        'id': '00:00:00:00:00:00:00:01',
+                        'interfaces': {
+                            '00:00:00:00:00:00:00:01:1': {
+                                'enabled': True,
+                                'id': '00:00:00:00:00:00:00:01:1',
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        status_disable = {
+            'network_status': {
+                'id': 'network_status',
+                'switches': {
+                    '00:00:00:00:00:00:00:01': {
+                        'dpid': '00:00:00:00:00:00:00:01',
+                        'enabled': False,
+                        'id': '00:00:00:00:00:00:00:01',
+                        'interfaces': {
+                            '00:00:00:00:00:00:00:01:1': {
+                                'enabled': False,
+                                'id': '00:00:00:00:00:00:00:01:1',
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        api = get_test_client(self.napp.controller, self.napp)
+        mock_storehouse.side_effect = [{}, status, status_disable]
+
+        # fail case
+        url = f'{self.server_name_url}/v3/restore'
+        response = api.get(url)
+        self.assertEqual(response.status_code, 404, response.data)
+        self.assertEqual(msg_expected, json.loads(response.data))
```

Comment:
Maybe we can test only http response code, because messages can be flexible.

Reply:
I agree. I implemented your suggestion. Thanks.